Options.py

```diff
@@ -1582,7 +1582,7 @@ def dump_player_options(multiworld: MultiWorld) -> None:
             }
             output.append(player_output)
             for option_key, option in world.options_dataclass.type_hints.items():
-                if issubclass(Removed, option):
+                if issubclass(Removed, option) or option.visibility == Visibility.none:
```

Comment:
```suggestion
                if option.visibility == Visibility.none:
```
Removed has Visibility.none so we can just skip checking specifically for it